src/Tizen.NUI/src/public/Rendering/Texture.cs

```diff
@@ -34,17 +34,15 @@ public class Texture : BaseHandle
         /// <param name="width">The width of the texture.</param>
         /// <param name="height">The height of the texture.</param>
         /// <since_tizen> 3 </since_tizen>
```

Comment:
```suggestion
        [EditorBrowsable(EditorBrowsableState.Never)]
```

Please mark it as hidden.